frontend/src/components/prefinderoverview.js

```diff
@@ -0,0 +1,74 @@
+import { Images } from "../data/image"
+import red from "../resources/like/Fire Heart.svg"
+import black from "../resources/like/Favorite.svg"
+import { useState,useEffect } from "react"
+function OverviewPrefinder(){
+    const [link1, setLink1] = useState(black)
+    const [link2, setLink2] = useState(black)
```

Comment:
Try optimising the code a bit. You can use an array with black as an initial values to all the elements.